apps/mobile/ios/Podfile

```
@@ -139,6 +139,45 @@ def download_and_compile_gopenpgp(version)
+          puts "No changes needed for app icon configuration"
+        end
+      else
+        puts "Warning: project.pbxproj file not found at #{project_file}"
+      end
+    else
+      puts "Xcode version is 26 or higher, no app icon configuration changes needed"
+    end
+  else
+    puts "Warning: Could not detect Xcode version"
+  end
+end
```

RuboCop (1.80.2)

[convention] Perceived complexity for `support_below_xcode26_app_icon` is too high. [9/8]

(Metrics/PerceivedComplexity)